dev/com.ibm.websphere.appserver.api.jaxrs20/bnd.bnd

```diff
@@ -17,6 +17,7 @@ Bundle-SymbolicName: com.ibm.websphere.appserver.api.jaxrs20
 
 Import-Package: com.ibm.websphere.jaxrs.providers.json4j, \
 		com.ibm.websphere.jaxrs20.multipart, \
+		com.ibm.websphere.jaxrs.monitor, \
```

Comment:
I think you will also need to export `com.ibm.websphere.monitor.jmx` - and I think both this and `com.ibm.websphere.jaxrs.monitor` should include `;resolution:=optional` so that those packages are not required to resolve this bundle.

Reply:
will do, thanks